compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1802,7 +1805,7 @@ impl Target {
         key!(exe_suffix);
         key!(staticlib_prefix);
         key!(staticlib_suffix);
-        key!(os_family = "target-family", optional);
+        key!(families = "target-family", list);
```

Comment:
You may want to support the old syntax for compatibility, this is an old and potentially popular option.